src/components/formik/FormSuccess.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import Style from './FormSuccess.module.css'
+const FormSubmitted = () => {
+    return (
+        <div>
+          <div className={Style.card}>
+            <div className={Style.Dcard}>
+              <i className="checkmark">✓</i>
+            </div>
+            <h1>Success</h1> 
+              <p>Your Form has been submitted sucessfully<br/> we'll be in touch shortly!</p>
```

Comment:
Could you change to:

```suggestion
              <p>Your form has been submitted successfully.</p>
              <p>We'll be in touch shortly!</p>
```

Some grammar/spelling fixes, and I think having two paragraphs feels more clear semantically.